src/solver/modelConverter/modelConverter.cpp

```diff
@@ -0,0 +1,211 @@
+}
+
+/**
+ * \brief Converts constraints from ModelParser::Model to ObjectModel::Constraint.
+ *
+ * \param model The ModelParser::Model object containing constraints.
+ * \return A vector of ObjectModel::Constraint objects.
+ */
+std::vector<Antares::Solver::ObjectModel::Port> convertPorts(
+  const Antares::Solver::ModelParser::Model& model)
+{
+    std::vector<Antares::Solver::ObjectModel::Port> ports;
+    for (const auto& port: model.ports)
+    {
+        // ports.emplace_back(Antares::Solver::ObjectModel::Port{port.name, port.type});
+    }
+    return ports;
+}
+
+std::vector<Antares::Solver::ObjectModel::Constraint> convertConstraints(
+  const Antares::Solver::ModelParser::Model& model)
+{
+    std::vector<Antares::Solver::ObjectModel::Constraint> constraints;
+    for (const auto& constraint: model.constraints)
+    {
+        constraints.emplace_back(Antares::Solver::ObjectModel::Constraint{
+          constraint.id,
+          Antares::Solver::ObjectModel::Expression{constraint.expression}});
+    }
+    return constraints;
+}
+
+/**
+ * \brief Converts models from ModelParser::Library to ObjectModel::Model.
+ *
+ * \param library The ModelParser::Library object containing models.
+ * \return A vector of ObjectModel::Model objects.
+ */
+std::vector<Antares::Solver::ObjectModel::Model> convertModels(
+  const Antares::Solver::ModelParser::Library& library)
+{
+    std::vector<Antares::Solver::ObjectModel::Model> models;
+    for (const auto& model: library.models)
+    {
+        Antares::Solver::ObjectModel::ModelBuilder modelBuilder;
+        std::vector<Antares::Solver::ObjectModel::Parameter> parameters = convertParameters(model);
+        std::vector<Antares::Solver::ObjectModel::Variable> variables = convertVariables(model);
+        std::vector<Antares::Solver::ObjectModel::Port> ports = convertPorts(model);
+        std::vector<Antares::Solver::ObjectModel::Constraint> constraints = convertConstraints(
+          model);
+
+        auto modelObj = modelBuilder.withId(model.id)
+                          .withObjective(Antares::Solver::ObjectModel::Expression{model.objective})
+                          .withParameters(std::move(parameters))
+                          .withVariables(std::move(variables))
+                          .withPorts(std::move(ports))
+                          .withConstraints(std::move(constraints))
+                          .build();
+        models.emplace_back(std::move(modelObj));
+    }
+    return models;
+}
+
+/**
+ * \brief Converts a ModelParser::Library object to an ObjectModel::Library object.
+ *
+ * \param library The ModelParser::Library object to convert.
+ * \return The corresponding ObjectModel::Library object.
+ */
+Antares::Solver::ObjectModel::Library convert(const Antares::Solver::ModelParser::Library& library)
+{
+    Antares::Solver::ObjectModel::LibraryBuilder builder;
+    std::vector<Antares::Solver::ObjectModel::PortType> portTypes = convertTypes(library);
```

Comment:
_Cosmetic_
About : 
```c++
std::vector<Antares::Solver::ObjectModel::PortType> portTypes = convertTypes(library);
```
Why not (instead) :
 ```c++
std::vector<Antares::Solver::ObjectModel::PortType> portTypes = convertTypes(library.port_types);
```
?
Maybe more natural. 
Note that this remark applies to multiple convert function in this file

Reply:
No particular reason